tests/test_auth.py

```diff
@@ -254,9 +254,14 @@ def test_jwt_decode_params(self, request_mock, jws_mock):
         }
         request_mock.post.return_value = post_json_mock
         self.backend.authenticate(request=auth_request)
-        calls = [
-            call('token', 'client_secret')
-        ]
+        if six.PY2:
+            calls = [
+                call('token', 'client_secret')
+            ]
+        else:
+            calls = [
+                call(b'token', b'client_secret')
+            ]
```

Comment:
I *could* use `smart_bytes` here too. 
I never know which is best (practice). In other words, this'd work too:
```python
calls = [
    call(smart_bytes('token'), smart_bytes('client_secret'))
]
```

Reply:
Just remove the `six` condition and use `django.utils.encoding.force_bytes`, unless you think this will receive a Django promise (e.g. returned by `ugettext_lazy()`). It's what `smart_bytes` calls eventually.